src/docs/integrations/jira/index.mdx

```diff
@@ -1,18 +1,27 @@
 ---
-title: "Jira Integration"
+title: 'Jira Integration'
 ---
 
-Create an [Atlassian Cloud Instance](http://go.atlassian.com/cloud-dev)
+Create an [Atlassian Cloud Developer Instance](http://go.atlassian.com/cloud-dev)
 
-Configure Jira settings.
-  1. Navigate to Jira settings (cog icon) > Apps > Manage apps.
-  2. Scroll to the bottom of the Manage apps page, and click Settings (1).
-  3. Select Enable development mode and Enable private listings (2), and click Apply.
+Atlassian offers a development-specific org option, different from simply having a personal Jira organization. Even if you already have a personal Jira org, it's worth creating a development org, because it will correctly set a number of necessary options for you automatically.
+
+Set Up ngrok
+
+(This is only necessary if you're developing locally.) [Download](https://ngrok.com/download) and [set up](https://ngrok.com/docs/getting-started/) ngrok, then start an ngrok server.
```

Comment:
I'm okay with leaving this in, but note that every integration that has instructions in the develop docs needs this to develop locally. It might be better served as it's own page as you mentioned, just feels a little strange to have it on one page and not the others.

Reply:
Yeah, once I move ngrok stuff from Notion to a real dev docs page, this (and any other integration) can just have a one-liner, along the lines of "if you're developing locally, start up ngrok" and link to the setup page.